python/invoke_release/tasks.py

```diff
@@ -1293,7 +1358,21 @@ def release(_, verbose=False, no_stash=False):
         _standard_output('Releasing {}...', MODULE_DISPLAY_NAME)
         _standard_output('Current version: {}', __version__)
 
-        release_version = _prompt('Enter a new version (or "exit"):').lower()
+        cl_header, cl_message, cl_footer = _prompt_for_changelog(verbose)
+        suggested_version = _suggest_version(__version__, _get_version_to_bump(cl_message))
+
+        instruction = None
+        if suggested_version:
+            instruction = _prompt(
+               'According to the changelog message the next version should be `{}`. '
+               'Do you want to proceed with the suggested version? (y/N)'.format(suggested_version)
```

Comment:
If we have an intelligent version suggestion, I think we should default to Y instead of N, but that's just, like, my opinion, dude. 😂 I'd really like to hear your thoughts on if/why it should be N like you have it here.

Reply:
I guess I just needed someone saying this out loud to me, haha. Thanks.